src/providers/details/tabs/Details/components/InventorySection/VmCountDetailsItem.tsx

```diff
@@ -0,0 +1,42 @@
+import type { FC } from 'react';
+import { DetailsItem } from 'src/modules/Providers/utils/components/DetailsPage/DetailItem';
+import { VirtualMachinesCell } from 'src/providers/components/VirtualMachinesCell';
+import { ProvidersResourceFieldId } from 'src/providers/utils/constants';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import type { InventoryDetailsItemProps } from './utils/types';
+
+const VmCountDetailsItem: FC<InventoryDetailsItemProps> = ({
+  helpContent,
+  inventory,
+  resource: provider,
+}) => {
+  const { t } = useForkliftTranslation();
+
+  const defaultHelpContent = t(`Number of virtual machines in provider.`);
+
+  return (
+    <DetailsItem
+      title={t('Virtual machines')}
+      helpContent={helpContent ?? defaultHelpContent}
+      crumbs={['Inventory', 'providers', provider?.spec?.type ?? '', '[UID]', 'vmCount']}
+      content={
+        <VirtualMachinesCell
+          data={{
+            inventory,
+            inventoryLoading: true,
+            permissions: undefined,
```

Comment:
Why declare undefined?

Reply:
I'm using the same inventory components (in this case `VirtualMachinesCell` component) for the providers list and for the details tab, so that the format+icon+hyper link will be the same.

 All those components require the `ProvderData` prop.
In this scenario, I don't need it  and don't have it except the inventory and provider field  so filed with undefineds

Reply:
so make it optional, no point in sending undefined